testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -61,6 +61,20 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-core</artifactId>
+            <scope>test</scope>
+            <type>test-jar</type>
+        </dependency>
+        <dependency>
```

Comment:
When running this:
```
cd testsuite/integration-arquillian/tests/base
mvn dependency:tree
```
I see that keycloak-core and keycloak.crypto.artifactId are already included as dependencies of this module. So it shouldn't be needed to explicitly include them again this way? Is it possible that you keep just the first dependency (keycloak-core test-jar) in this PR, but remove the other two dependencies `keycloak-core` and `keycloak.crypto.artifactId` from this PR?